hm_3/hm_03.py

```diff
@@ -0,0 +1,25 @@
+# task_1
+string = "I love Python" * 42
+print(string)
+# task_2
+age_in_month = 24 * 12 + 10
+print(age_in_month)
+# task_3
+age_in_years = age_in_month // 12
+print(age_in_years)
+# task_4
+name = "Lolita"
+my_age = f"My name is {name}. I'm {age_in_years} years old"
+print(my_age)
+# task_5
+i = 1
+print(2 != i)  # True
+print(4 == i)  # False
+print(1 >= i)  # True
+print(5 < i)   # False
+print(8 > i)   # True
+# task_6gi
+a = 2
+b = 5
+c = 6
+d = a ** (a + c)
```

Comment:
`d` variable should be a string and it should be a result of concatenation variables `a, b, c` transformed to strings